workloads/network-perf/run_multus_network_tests_fromgit.sh

```diff
@@ -135,6 +140,17 @@ if [ "$uperf_state" == "1" ] ; then
   exit 1
 fi
 
+if [[ ${COMPARE} == "true" ]] ; then
+  baseline_uperf_uuid=${_baseline_multus_uuid}
+  if [ "$_metadata_collection" == true ] ; then
```

Comment:
We should do a json query to extract uuid, it's simpler and the code would be much more stable.

Reply:
agreed